docs/fundamentals/documentarray/evaluation.md

```diff
@@ -3,7 +3,7 @@
 After you get `.matches` from the last chapter, you can easily evaluate matches against the groundtruth via {meth}`~docarray.array.mixins.evaluation.EvaluationMixin.evaluate`.
```

Comment:
```suggestion
After you get `.matches`, you can evaluate matches against the groundtruth via {meth}`~docarray.array.mixins.evaluation.EvaluationMixin.evaluate`.
```
I know this has nothing to do with this PR, but let's get away form treating our docs like a book